components/km271_wifi/number.py

```diff
@@ -50,8 +56,42 @@
                 cv.Optional(CONF_MIN_VALUE, default=10): cv.float_,
                 cv.Optional(CONF_STEP, default=0.5): cv.positive_float,
             }),
-            
-
+            cv.Optional(CONF_HC1_ROOMT_TARGET_OFFSET): number.NUMBER_SCHEMA.extend({
+                cv.GenerateID(): cv.declare_id(BuderusParamNumber),
+                cv.Optional(CONF_MAX_VALUE, default=5): cv.float_,
+                cv.Optional(CONF_MIN_VALUE, default=-5): cv.float_,
+                cv.Optional(CONF_STEP, default=1): cv.float_,
+            }),
+            cv.Optional(CONF_HC1_FT_TARGET_MAX): number.NUMBER_SCHEMA.extend({
+                cv.GenerateID(): cv.declare_id(BuderusParamNumber),
+                cv.Optional(CONF_MAX_VALUE, default=90): cv.float_,
+                cv.Optional(CONF_MIN_VALUE, default=20): cv.float_,
+                cv.Optional(CONF_STEP, default=1): cv.positive_float,
+            }),
+            cv.Optional(CONF_HC2_DESIGN_TEMP): number.NUMBER_SCHEMA.extend({
+                cv.GenerateID(): cv.declare_id(BuderusParamNumber),
+                cv.Optional(CONF_MAX_VALUE, default=90): cv.float_,
+                cv.Optional(CONF_MIN_VALUE, default=30): cv.float_,
+                cv.Optional(CONF_STEP, default=1): cv.positive_float,
+            }),
+            cv.Optional(CONF_HC2_ROOMT_TARGET_DAY): number.NUMBER_SCHEMA.extend({
+                cv.GenerateID(): cv.declare_id(BuderusParamNumber),
+                cv.Optional(CONF_MAX_VALUE, default=30): cv.float_,
+                cv.Optional(CONF_MIN_VALUE, default=10): cv.float_,
+                cv.Optional(CONF_STEP, default=0.5): cv.positive_float,
+            }),
+            cv.Optional(CONF_HC2_ROOMT_TARGET_OFFSET): number.NUMBER_SCHEMA.extend({
```

Comment:
In the manual this is called Raumtemperatur-Offset, so "TARGET" is not needed here, imho. 
https://www.manualslib.de/manual/15526/Buderus-Logamatic-2107.html?page=41#manual

Reply:
I just called it that way to be consistent with CONF_HC2_ROOMT_TARGET_DAY, but we can change if you prefer it that way.

Reply:
I'd prefer it without TARGET.

Reply:
and also in the yaml and this onces
config_heating_circuit_2_room_target_temperature_offset
right?

Reply:
I would keep CONF_HC2_ROOMT_TARGET_DAY: This is the target room temperature (Raumsolltemperatur).
But for the OFFSET-setting there is no TARGET in the manual.

Reply:
ok, lets make it so